knowledge-base/font-icons-not-rendering.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Font Icons do not render in Telerik UI for Blazor 4.6
+description: Font Icons do not render in Telerik UI for Blazor 4.6
+type: troubleshooting
+page_title: Font Icons do not Render in Telerik UI for Blazor 4.6.0
+slug: icon-kb-font-icons-not-rendering
+position: 
+tags: telerik, blazor, fonticon, font, icon
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>4.6</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>Progress® Telerik® UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+After upgrading to the 4.6.0 version of the Telerik UI for Blazor suite the Font Icons in my application are not longer showing (rendering). 
+
+## Possible Cause
+
+Each Telerik UI for Blazor version has a dependency on a specific version of the Kendo UI Themes. As part of the [7.0.0 version of the Kendo UI Themes](https://github.com/telerik/kendo-themes/blob/develop/CHANGELOG.md#breaking-changes), the Font Icons were detached from the themes distribution.
+
+The 7.0.0 version of the Kendo UI Themes is a dependency in Telerik UI for Blazor 4.6.0; thus, the font icons are missing.
+
+## Solution
```

Comment:
No need to duplicate the examples for index.html and _Host.cshtml. It may be easier to just mention that the new stylesheet should be added where the theme is.

Reply:
Again, foolproof to prevent any misunderstandings. I didi this to be on the safe-side due to previous mishaps with Icons (4.0.0). If you believe that it is redundant I can remove it.

Reply:
It doesn't make sense to encourage users to click on a tab and see the same code. I would clarify the required file before the snippet.

Reply:
Remove the duplicates